src/main/java/frc/robot/subsystems/leds/LEDSubsystem.java

```diff
@@ -38,20 +44,97 @@ public void periodic() {
         // Toggle the blink boolean every duration to swap the LEDs between the driver piece color
         // and the blink color.
         if (blinkTimer.advanceIfElapsed(BLINK_DURATION_SECONDS)) blinking = !blinking;
-        ledStrip.setSolidColor(blinking ? BLINK_COLOR : color);
+        if(manual && continuous){
+            //if manual and continuous set the bottom to color
+            ledStrip.updateContinuousColor(color);
+            ledStrip.setContinuousColor();
+        } else if(manual){
+            //if manual and not continuous fill entire strip with color
+            ledStrip.fillContinuousColor(color);
+            ledStrip.setContinuousColor();
+        } else {
+            setColorPulse();
+            //setTwoColor()
+        }
```

Comment:
we should not set the LEDs in both the periodic and in the `setRGB`/`setHSV` methods.